app/experimenter/experiments/forms.py

```diff
@@ -1015,7 +1037,7 @@ class ExperimentReviewForm(ExperimentConstants, ChangeLogMixin, forms.ModelForm)
     # Optional
     review_advisory = forms.BooleanField(
         required=False,
-        label="Lightning Advisory (Optional)",
+        label="Lightning Advisory",
```

Comment:
Would this be a little confusing? 
From what I understand from https://github.com/mozilla/experimenter/pull/1998/files#diff-b75d2c1707450c0ad27a64a54719a0bbR1106, rollouts requires lightning advisory as a hard requirement while the other types it's still technically optional but it's in the required list?

Reply:
I feel like it's confusing either way, if it's in the required list but says optional, and now it might be required it might not, so I was banking on letting the process sort it out and we might find a new way to represent it rather than having custom logic to shift the label back and forth.